src/translations/obconf-qt_zh_CN.ts

```diff
@@ -0,0 +1,3 @@
+Name[zh_CN]=Openbox 设置
```

Comment:
Shouldn't it be the .desktop file? (you've named it .ts)

Reply:
Sorry, I will fix it later.